blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -99,12 +139,145 @@ const LargePromoItem = ({ customFields, arcSite }) => {
+		new Date(content?.display_date),
+		dateTimeFormat,
+		language,
+		timeZone
+	);
+
+	const editableDescription = content?.description
+		? editableContent(content, "description.basic")
+		: {};
+
+	return (
+		<HeadingSection>
+			<Grid as="article" className={BLOCK_CLASS_NAME}>
+				{showImage ? (
+					<MediaItem
+						className={`${BLOCK_CLASS_NAME}__media`}
+						{...searchableField("imageURL")}
+						suppressContentEditableWarning
+					>
+						<Conditional
+							component={Link}
+							condition={content?.websites?.[arcSite]?.website_url}
+							href={formatURL(content?.websites?.[arcSite]?.website_url)}
+							onClick={registerSuccessEvent}
+							assistiveHidden
+						>
+							{playVideoInPlace ? (
+								<Video
+									aspectRatio={aspectRatio}
+									className={`${BLOCK_CLASS_NAME}__video`}
+									embedMarkup={customFields?.content?.embed_html}
+									viewportPercentage={viewportPercentage}
+								/>
+							) : (
+								<Image alt={content?.headlines?.basic || null} src={targetImage} searchableField />
+							)}
+							{showImageOrVideoLabel ? (
+								<div className={`${BLOCK_CLASS_NAME}__icon_label`}>
+									<Icon name={showVideoLabel ? "Play" : "Instagram"} fill="#FFFFF" />
+									<span className={`${BLOCK_CLASS_NAME}__label`}>{imageOrVideoLabelText}</span>
+								</div>
+							) : null}
+						</Conditional>
+					</MediaItem>
+				) : null}
+				{showOverline || showHeadline || showDescription || showByline || showDate ? (
+					<Grid className={`${BLOCK_CLASS_NAME}__text`}>
+						{showOverline && (url || text) ? (
+							<Overline href={url ? formatURL(url) : null}>{text}</Overline>
+						) : null}
+						<Stack>
+							{showHeadline && content?.headlines?.basic ? (
+								<Heading>
+									<Conditional
+										component={Link}
+										condition={content?.websites?.[arcSite]?.website_url}
+										href={formatURL(content?.websites?.[arcSite]?.website_url)}
+										onClick={registerSuccessEvent}
+									>
+										{content?.headlines?.basic}
+									</Conditional>
+								</Heading>
+							) : null}
+							{showDescription ? (
+								<Paragraph suppressContentEditableWarning {...editableDescription}>
+									{content?.description?.basic}
+								</Paragraph>
+							) : null}
+							{showByline || showDate ? (
+								<div className={`${BLOCK_CLASS_NAME}__meta`}>
+									{showByline && bylineNodes?.length > 0 ? (
+										<Attribution>
+											<span className={`${BLOCK_CLASS_NAME}__by`}>
+												{phrases.t("global.by-text")}
+											</span>{" "}
+											<span className={`${BLOCK_CLASS_NAME}__names`}>{bylineNodes}</span>
+										</Attribution>
+									) : null}
+									{showByline && showDate ? <Separator /> : null}
+									{showDate && content?.display_date ? (
+										<DateDisplay dateTime={content.display_date} dateString={displayDate} />
+									) : null}
+								</div>
+							) : null}
```

Comment:
As suggested in the Slack thread:
```suggestion
							{showByline || showDate ? (
								<Attribution>
									<Join separator={Separator}>
										{showByline ? (
											<Join separator={() => " "}>
												{phrases.t("global.by-text")}
												{bylineNodes}
											</Join>
										) : null}
										{showDate ? (
											<DateDisplay dateTime={displayDate} dateString={formattedDate} />
										) : null}
									</Join>
								</Attribution>
							) : null}
```
`<Attribution />` can be used for the wrapper (where "meta" is now) for styling
I created a `<Join />` component for handling the separator and it's conditions (also used here to fill in a space)

Reply:
has this been addressed @malavikakoppula ?

Reply:
Yes Jack, it's updated